impl/src/bazel.rs

```diff
@@ -120,15 +120,15 @@ pub fn is_bazel_supported_platform(target: &str) -> (bool, bool) {
   (is_supported, matches_all)
 }
 
-/** Maps a Rust cfg target to a Bazel supported triples.
+/** Maps a Rust cfg or triple target to Bazel supported triples.
  *
  * Note, the Bazel triples must be defined in:
  * https://github.com/bazelbuild/rules_rust/blob/master/rust/platform/platform.bzl
  */
 pub fn get_matching_bazel_triples(target: &str) -> Result<Vec<String>> {
   let target_exp = match target.starts_with("cfg(") {
     true => target.to_owned(),
-    false => format!("cfg(target=\"{}\")", target),
+    false => format!("cfg(target = \"{}\")", target),
```

Comment:
This is just for consistency with the `is_bazel_supported_platform` function above.